.github/workflows/e2e.yml

```diff
@@ -22,17 +22,17 @@ jobs:
   prepare-env-job:
     runs-on: ubuntu-latest
     outputs:
-      short_sha: ${{ steps.prepare.outputs.short_sha }}
+      version_suffix: ${{ steps.prepare.outputs.version_suffix }}
       pr_title: ${{ steps.prepare.outputs.pr_title }}
       oc_project: ${{ steps.prepare.outputs.oc_project }}
       test_pr_base_branch: ${{ steps.prepare.outputs.test_pr_base_branch }}
     steps:
       - id: prepare
         run: |
-          echo "::set-output name=short_sha::${GITHUB_SHA::7}"
-          echo "::set-output name=pr_title::operator test-e2e-operator (0.0.7-${GITHUB_SHA::7})"
-          echo "::set-output name=oc_project::test-e2e-${GITHUB_SHA::7}"
-          echo "::set-output name=test_pr_base_branch::${GITHUB_SHA::7}-base"
+          echo "::set-output name=version_suffix::${{ github.run_number }}${{ github.run_attempt }}"
+          echo "::set-output name=pr_title::operator test-e2e-operator (0.0.8-${{ github.run_number }}${{ github.run_attempt }})"
+          echo "::set-output name=oc_project::test-e2e-${{ github.run_number }}${{ github.run_attempt }}"
+          echo "::set-output name=test_pr_base_branch::${{ github.run_number }}${{ github.run_attempt }}-base"
```

Comment:
According to the docs, both of these values start at 1 and increment. So isn't this problematic when there are 10 or more run attempts? E.g.

```bash
0.0.8-11   # Run 1, attempt 1
0.0.8-12   # Run 1, attempt 2
0.0.8-13   # Run 1, attempt 3
...
0.0.8-110  # Run 1, attempt 10
0.0.8-21   # Run 2, attempt 1
```

When sorted by semver, 0.0.8-110 is going to be considered a replacement of 0.0.8-21 by `opm`.

Reply:
I don't think so. Because when I started using `run_number` instead of `run_id` and the opm worked fine at that point even though the `run_id` was a higher number than `run_number`.

Reply:
Adding a dash may even be sufficient (according to [this](https://semvercompare.azurewebsites.net) tool).

```bash
0.0.8-1-1   # Run 1, attempt 1
...
0.0.8-1-10  # Run 1, attempt 10
0.0.8-2-1   # Run 2, attempt 1
```

Reply:
Nevermind; adding a dash isn't good enough. As soon as the number of digits in the `run_number` goes above 9 it won't sort as desired.

Reply:
Here is the suggestion: `0.0.{run_number}-{run_attempt}`. It always be increasing.

Reply:
That works well! Our run numbers are already much higher that 7 or 8. It shouldn't cause any conflicts with the stage index images.